src/reconcile/rec_write.c

```diff
@@ -684,6 +684,7 @@ __rec_init(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags, WT_SALVAGE_COO
 
     r->wrapup_checkpoint = NULL;
     r->wrapup_checkpoint_compressed = false;
+    memset(&r->wrapup_checkpoint_block_meta, 0, sizeof(r->wrapup_checkpoint_block_meta));
```

Comment:
Could use WT_CLEAR

Reply:
Thanks—it makes sense that we would have a macro for this!